src/doc/trpl/unsafe.md

```diff
@@ -49,32 +49,15 @@ and mutable (`&mut`) references have some aliasing and freezing
 guarantees, required for memory safety.
 
 In particular, if you have an `&T` reference, then the `T` must not be
-modified through that reference or any other reference. There are some
-standard library types, e.g. `Cell` and `RefCell`, that provide inner
-mutability by replacing compile time guarantees with dynamic checks at
-runtime.
+modified through that reference or any other reference or pointer and must not
+alias with any `&mut` reference. There are some standard library types, e.g.
```

Comment:
Distinguishing between pointers and references seems a bit strange and overly verbose. It seems like we should just use a single word to refer to all pointer types. The language reference refers to them as 'pointers', but I've noticed in the past that some people in the Rust community dislike the P-word.